bBridgeAPISDK/Common/HttpRequester.cs

```diff
@@ -51,7 +68,6 @@ public HttpRequester(Uri baseAdress, IAuthorizer authorizer = null)
                     {
                         if (responseTask.Result.IsSuccessStatusCode)
```

Comment:
if `IsSuccessStatusCode` check that status code is 2xx then next condition is unreachable